projects/wave-core-new/src/lib/operators/dialogs/column-range-filter/column-range-filter.component.ts

```diff
@@ -0,0 +1,251 @@
+import {Component, OnInit, ChangeDetectionStrategy, OnDestroy} from '@angular/core';
+import {ResultTypes} from '../../result-type.model';
+import {FormArray, FormBuilder, FormGroup, Validators} from '@angular/forms';
+import {Observable, of, ReplaySubject, Subscription} from 'rxjs';
+import {ProjectService} from '../../../project/project.service';
+import {map, mergeMap} from 'rxjs/operators';
+import {Layer, VectorLayer} from '../../../layers/layer.model';
+import {VectorLayerMetadata} from '../../../layers/layer-metadata.model';
+import {VectorColumnDataType, VectorColumnDataTypes} from '../../datatype.model';
+import {WorkflowDict} from '../../../backend/backend.model';
+import {ClusteredPointSymbology, PointSymbology} from '../../../layers/symbology/symbology.model';
+import {colorToDict} from '../../../colors/color';
+import {RandomColorService} from '../../../util/services/random-color.service';
+import {ColumnRangeFilterDict} from '../../../backend/operator.model';
+
+@Component({
+    selector: 'wave-column-range-filter',
+    templateUrl: './column-range-filter.component.html',
+    styleUrls: ['./column-range-filter.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ColumnRangeFilterComponent implements OnInit, OnDestroy {
+    readonly inputTypes = ResultTypes.VECTOR_TYPES;
+
+    attributes$ = new ReplaySubject<Array<string>>(1);
+
+    form: FormGroup;
+
+    attributeError: boolean = false;
```

Comment:
Type string trivially inferred from a string literal, remove type annotation

This should be shown by eslint in your ide though

Reply:
removed type annotation